CHANGELOG.md

```
@@ -54,6 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - (x/async) Task can return errors instead of outputs, one of "output" or "error" fields will be set
 - (x/async) Add PruneTaskTimeout param. After a task is completed (i.e. has a result) it will be permanently deleted from the appdb after this timeout elapses.
 - (x/async) Add Plugin's timeout param. This is the maximum time a solver has for submitting the result before the Task is marked as errored automatically.
+- (x/crisis) Remove deprecated x/crisis module. See https://github.com/cosmos/cosmos-sdk/pull/23722. Will be removed in Cosmos SDK v0.54.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)